src/pages/CreateBoard/index.tsx

```diff
@@ -5,35 +5,47 @@ import theme from 'styles/theme';
 import Input from 'components/Input';
 import UploadIcon from 'assets/UploadIcon';
 import { useState } from 'react';
-import instance from 'apis/httpClient';
+import useImageUpload from 'hooks/useImageUpload';
+import { useMutation, useQueryClient } from 'react-query';
+import { createBoard } from 'apis/api';
+import { useNavigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
 
 const CreateBoard = () => {
-  const [imageUrl, setImageUrl] = useState<string>('');
+  const [name, setName] = useState<string>('');
+  const [content, setContent] = useState<string>('');
+  const { imageUrl, handleImageChange } = useImageUpload();
+  const router = useNavigate();
 
-  const handleImageChange = async (e: React.ChangeEvent<HTMLInputElement>) => {
-    if (e.target.files && e.target.files.length > 0) {
-      const selectedFile = e.target.files[0];
-      setImageUrl(URL.createObjectURL(selectedFile));
+  const onChange = (
+    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+  ) => {
+    const {
+      target: { name, value },
+    } = e;
 
-      const formData = new FormData();
-      formData.append('image', selectedFile);
+    if (name === 'title') {
+      setName(value);
+    }
+
+    if (name === 'content') {
+      setContent(value);
+    }
+  };
```

Comment:
```suggestion
    if (name === 'title') {
      setName(value);
      returnl
    }

    if (name === 'content') {
      setContent(value);
      return;
    }
  };
```

얼리 리턴을 사용하지않게된다면 첫번쨰의 조건문에 걸리게되어도 불필요하게 두번째 조건문도 검사하게됩니다 불필요한 검사는 없애주는게 좋을것같아용

Reply:
아 이런 방법이 있었군요!